modules/bitbar_devicepool/manifests/base.pp

```diff
@@ -38,11 +38,16 @@
     ensure => 'present',
   }
 
+  # disable login for departed users
```

Comment:
good idea to ensure users cannot login and are gone. (considering that just removing user creation could miss something)